src/ui/MessageContent/index.tsx

```diff
@@ -175,6 +175,16 @@ export default function MessageContent(props: MessageContentProps): ReactElement
     && message.myFeedbackStatus !== SbFeedbackStatus.NOT_APPLICABLE;
   const isFeedbackEnabled = config?.groupChannel?.enableFeedback && isFeedbackMessage;
   const feedbackMessageClassName = isFeedbackEnabled ? 'sendbird-message-content__feedback' : '';
+  /**
+   * For TemplateMessage, do not display:
+   *   - in web view:
+   *     - message menu
+   *     - reaction menu
+   *     - reply in thread
+   *   - in mobile view:
+   *     - bottom sheet on long click
+   */
```

Comment:
```ts
const shouldVisibleMessageMenu = isNotTemplateMessage && ~
const shouldVisibleReactionMenu = isNotTemplateMessage && ~
const shouldVisibleMobileBottomSheet = isNotTemplateMessage && ~
```

This is just a simple suggestion. It's okay if it's not implemented.
How about managing the grouping related to 'visible' as a variable? If certain values or conditions change later on, it might create synergy along with comments

Reply:
ooh yet I agree that would clear up any confusion that might arise. However, let's keep this in backlog for later because this might need multiple back and forth comunication which will take more time than we have now~

Reply:
actually let me push this commit fast

Reply:
this was simpler than i thought, done!